randomize/plugin.py

```diff
@@ -19,8 +19,37 @@
 from nose.util import isclass, isgenerator, transplant_func, transplant_class
 import random
 import unittest
+from random import getrandbits
+from nose.tools import nottest
 
 log = logging.getLogger(__name__)
+CLASS_SPECIFIC_RANDOMIZE_TESTS_FIELD_NAME = 'randomize_tests_seed'
+
+
+@nottest
```

Comment:
I'm confused: Do you mean when you test your project it includes this as something that gets tested?

The reason I ask is because I'm striving to keep the code coverage for this project high. I'm willing to write a test for this method, but I wanted to understand if there was something going on here that warranted not having a test.